admiral/pkg/clusters/virtualservice_routing.go

```diff
@@ -16,6 +16,10 @@ import (
 	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
```

Comment:
Let's move this here: https://github.com/istio-ecosystem/admiral/blob/master/admiral/pkg/util/constants.go?